sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpReceiver.cs

```diff
@@ -33,17 +33,27 @@ namespace Azure.Messaging.ServiceBus.Amqp
     internal class AmqpReceiver : TransportReceiver
 #pragma warning restore CA1001 // Types that own disposable fields should be disposable
     {
-        /// <summary>Indicates whether or not this instance has been closed.</summary>
-        private bool _closed;
-
         /// <summary>
-        /// Indicates whether or not this receiver has been closed.
+        /// Indicates whether or not this receiver has been closed by the user.
         /// </summary>
         ///
         /// <value>
         /// <c>true</c> if the receiver is closed; otherwise, <c>false</c>.
         /// </value>
-        public override bool IsClosed => _closed;
+        public override bool IsReceiverClosedByUser => _receiverClosedByUserByUser;
```

Comment:
Maybe consider something like `WasClosedExplicitly` or similar?   It's not particularly important since it's an internal property... it just reads a bit oddly to me.